galpy/potential/potential_c_ext/RotateAndTiltWrapperPotential.c

```diff
@@ -36,6 +49,7 @@ void RotateAndTiltWrapperPotentialxyzforces(double R, double z, double phi,
     *(args + 3)= z;
     //now get the forces in R, phi, z in the aligned frame
     rotate(&x,&y,&z,rot);
+    apply_offset(&x,&y,&z,offsetSet,offset);
```

Comment:
I think we can just inline this here, in another branch I've also remove the `rotate` functions from this file:

```suggestion
    if ( offsetSet ) {
        x+= *offset;
        y+= *(offset+1);
        z+= *(offset+2);
    }
```